examples/gno.land/r/gnoland/users/types.gno

```diff
@@ -0,0 +1,24 @@
+package users
+
+import (
+	"gno.land/p/demo/ufmt"
+	susers "gno.land/p/sys/users"
+)
+
+// User extends the p/sys/users struct for more custom data
+type User struct {
+	base         *susers.User
+	Bio          string
```

Comment:
the `r/demo/profile` realm already have this field, do we really want it there too?

Reply:
Well, we'd want the `r/gnoland/users` to be the main "user app" for registration, with a possible v2, v3, etc.

`r/demo/profile` does not seem generic and well-built enough to utilize it in this realm. That's why I'd like a simple extension to the `p/sys/users`. WDYT?

Reply:
The real important thing for me is that there is a SSOT for an user's "Bio" field, whether it's in `profile` or `users`.

For me it make more sense to keep it in the `profile` realm and give the `users` realm only the responsibility to handle usernames. Why not put also "display name" and "avatar" in the `users` realm?

An ui or gnoweb render can merge the information from both sources. That's what Teritori's `cockpit` realm (which is a kind of user profile) already does for example.

Reply:
This makes sense; let's use `r/demo/profile` as a place which multiple apps can look up secondary information about a user. We might improve/move `profile` to a different namespace down the line though.